### PR TITLE
Use single zone count information

### PR DESCRIPTION
**Pull request by stefansjfw**
## PR Checklist
* [] Applies to #xxx
* [] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA
* [] Tests added/passed
* [] Requires documentation to be updated
* [] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

### CODE REVIEW

File: src/modules/fancyzones/tests/UnitTests/JsonHelpers.Tests.cpp [hunk 1]
```diff
@@ -520,60 +520,24 @@ namespace FancyZonesUnitTests
     };
 
     TEST_CLASS(ZoneSetDataUnitTest){
-        TEST_METHOD(ToJsonCustom)
```
Comment (SeraphimaZykova) [inline src/modules/fancyzones/tests/UnitTests/JsonHelpers.Tests.cpp:523]: Why these tests were removed?

**Review by SeraphimaZykova — APPROVED** [review]

File: src/modules/fancyzones/tests/UnitTests/JsonHelpers.Tests.cpp (hunk 1 above)
Comment (stefansjfw) [reply src/modules/fancyzones/tests/UnitTests/JsonHelpers.Tests.cpp:523]: Without zone-count value in json string, this test is basically the same as `ToJsonGeneral` test.. So we would have 2 tests testing same thing.

File: src/modules/fancyzones/tests/UnitTests/ZoneWindow.Spec.cpp [hunk 2]
```diff
@@ -108,7 +108,7 @@ namespace FancyZonesUnitTests
             Assert::IsFalse(std::filesystem::exists(activeZoneSetTempPath));
 
             const auto type = JSONHelpers::ZoneSetLayoutType::Columns;
-            const auto expectedZoneSet = JSONHelpers::ZoneSetData{ CreateGuidString(), type, 5 };
+            const auto expectedZoneSet = JSONHelpers::ZoneSetData{ CreateGuidString(), type};
```
Comment (vldmr11080) [inline src/modules/fancyzones/tests/UnitTests/ZoneWindow.Spec.cpp]: Space after `type` before `}'`

File: src/modules/fancyzones/tests/UnitTests/ZoneWindow.Spec.cpp [hunk 3]
```diff
@@ -263,7 +261,7 @@ namespace FancyZonesUnitTests
 
             const ZoneSetLayoutType type = ZoneSetLayoutType::Custom;
             const auto customSetGuid = CreateGuidString();
-            const auto expectedZoneSet = ZoneSetData{ customSetGuid, type, 5 };
+            const auto expectedZoneSet = ZoneSetData{ customSetGuid, type};
```
Comment (vldmr11080) [inline src/modules/fancyzones/tests/UnitTests/ZoneWindow.Spec.cpp]: Space after `type` before `}'`

**Review by vldmr11080 — APPROVED** [review]

**Pull request merged**